be/src/storage/rowset/page_io.cpp

```diff
@@ -224,21 +244,21 @@ Status PageIO::read_and_decompress_page(const PageReadOptions& opts, PageHandle*
         memcpy(decompressed_body.data + decompressed_body.size, page_slice.data + body_size, footer_size + 4);
         // free memory of compressed page
         page = std::move(decompressed_page);
-        page_slice = Slice(page.get(), footer->uncompressed_size() + footer_size + 4);
+        page_slice = Slice(page->data(), decompressed_page_size);
         opts.stats->uncompressed_bytes_read += page_slice.size;
     } else {
-        opts.stats->uncompressed_bytes_read += body_size;
+        opts.stats->uncompressed_bytes_read += page_slice.size;
     }
```

Comment:
Why replace the `body_size` with `page_slice.size`? It seems the `page_slice.size` also contains footer size if no need to decompress page, and it may not be the real uncompressed page size.

Reply:
I changed this because the preceding code also includes the footer, it should be consistent

```
        page_slice = Slice(page.get(), footer->uncompressed_size() + footer_size + 4);
```